roles/octavia/tasks/generate_resources.yml

```diff
@@ -152,3 +152,89 @@
     cloud: atmosphere
     image: "{{ octavia_amphora_image_name }}"
   register: _octavia_amphora_image
+
+- name: Deploy Amphora jumpbox
+  run_once: true
+  block:
+    - name: Create amphora ssh key
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          - apiVersion: secretgen.k14s.io/v1alpha1
+            kind: SSHKey
+            metadata:
+              name: amphora-ssh-key
+              namespace: openstack
+            spec: {}
+
+    - name: Wait until ssh key is reconciled
+      kubernetes.core.k8s_info:
+        api_version: apps/v1
+        kind: SSHKey
+        name: amphora-ssh-key
+        namespace: openstack
+        wait_sleep: 1
+        wait_timeout: 60
+        wait: true
+        wait_condition:
+          type: ReconcileSucceeded
+          status: true
+
+    - name: Create jumpbox deployment
+      kubernetes.core.k8s:
+        state: present
+        definition:
+          - apiVersion: apps/v1
+            kind: Deployment
+            metadata:
+              name: amphora-jumpbox
+              namespace: openstack
+              labels:
+                app: octavia
+                components: amphora-jumpbox
+            spec:
+              replicas: 1
+              selector:
+                matchLabels:
+                  app: octavia
+                  components: amphora-jumpbox
+              template:
+                metadata:
+                  labels:
+                    app: octavia
+                    components: amphora-jumpbox
+                spec:
+                  containers:
+                    - name: jumpbox
+                      image: "{{ atmosphere_images['octavia_jumpbox'] | vexxhost.kubernetes.docker_image('ref') }}"
+                      volumeMounts:
+                        - name: sshkey
+                          mountPath: /sshkey
+                          readOnly: true
+                  nodeSelector:
+                    openstack-control-plane: enabled
+                  volumes:
+                    - name: sshkey
+                      secret:
+                        secretName: amphora-ssh-key
+                        defaultMode: 0400 # noqa: yaml[octal-values]
```

Comment:
Instead of adding a new deployment, why don't we just simply use the `volumeMounts` feature and add it into the `octavia-worker` pod?  I think that's an easier and more straight forward approach (for now).